packages/css/src/components/header/header.scss

```diff
@@ -4,83 +4,201 @@
  */
 
 @import "../../common/breakpoint";
+@import "../../common/text-rendering";
 
 .ams-header {
+  display: grid;
+  padding-block: var(--ams-header-padding-block);
+  padding-inline: var(--ams-grid-padding-inline);
+}
+
+.ams-header__brand-section {
   align-items: center;
+  align-self: start;
+  column-gap: var(--ams-header-brand-section-column-gap);
   display: flex;
-  flex-wrap: wrap;
-  padding-block: var(--ams-header-padding-block);
-  row-gap: 1.5rem;
+  grid-area: 1 / 1;
+}
 
-  @media screen and (min-width: $ams-breakpoint-wide) {
-    column-gap: var(--ams-header-column-gap);
-    flex-wrap: nowrap;
-  }
+.ams-header__brand-section--hidden {
+  opacity: 0%;
+  user-select: none;
 }
 
 .ams-header__logo-link {
-  flex: none;
   outline-offset: var(--ams-header-logo-link-outline-offset);
 }
 
-.ams-header__links {
-  display: none;
+/* TODO Remove after updating Header line heights in DES-973. */
+.ams-heading.ams-header__app-name {
+  line-height: 1.335;
```

Comment:
I'm not sure it's supposed to? The intention here was to match the line height suggested for level 5 [here](https://gemeente-amsterdam.atlassian.net/browse/DES-973) (which is 1.35 btw, I'll change that).

Can you weigh in on what's supposed to happen here @VincentSmedinga?

Reply:
When the app name breaks to a second line it looks misaligned next to the logo, I suggested a while ago that we could match it to the logo. But since then the logo is scaling so that would make it harder.